gfxvs-ai-agent/public/.well-known/ai-plugin.json

```diff
@@ -0,0 +1,128 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "GFXvs API Mainnet",
+        "description": "API for interacting with GFXvs operations including gfxvs art upload.",
+        "version": "1.0.0"
+    },
+    "servers": [
+        {
+            "url": "https://gfxvsaiagent-dvb6a2hnd4araccq.eastus-01.azurewebsites.net"
```

Comment:
Change to `gfxvs.com`

Reply:
We have maintained this agent at https://gfxvsaiagent-dvb6a2hnd4araccq.eastus-01.azurewebsites.net  and also registered with Bitte. The reference chat link - https://wallet.bitte.ai/smart-actions/share/O9nKAfnxQhSetKkz0ZXbA